docs/conf.py

```diff
@@ -82,4 +82,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
to remove a warning when building locally

Reply:
oh ty